opentelemetry/proto/logs/v1/logs.proto

```diff
@@ -116,6 +116,23 @@ message LogRecord {
   // Value of 0 indicates unknown or missing timestamp.
   fixed64 time_unix_nano = 1;
 
+  // Time when the event was observed by the collection system.
+  // For events that originate in OpenTelemetry (e.g. using OpenTelemetry Logging SDK)
+  // this timestamp is typically set at the generation time and is equal to Timestamp.
+  // For events originating externally and collected by OpenTelemetry (e.g. using
+  // Collector) this is the time when OpenTelemetry's code observed the event measured
+  // by the clock of the OpenTelemetry code. This field SHOULD be set once the event is
```

Comment:
Can we strengthen the requirement here? My understanding is that this field is intended to be required.
```suggestion
  // by the clock of the OpenTelemetry code. This field MUST be set once the event is
```

Reply:
Yes, I think that's a good suggestion. Changed.